assets/styles/components/specials/_pullquotes.scss

```diff
@@ -44,14 +44,15 @@ aside,
 
 .pullquote {
   @extend %pullquote;
-  margin: $pullquote-margin-top 0 $pullquote-margin-bottom;
+  margin: if-map-get($pullquote-margin-top, $type) auto if-map-get($pullquote-margin-bottom, $type);
+  width: if-map-get($pullquote-width, $type);
 }
 
 .pullquote-left,
 .pullquote--left {
   @extend %pullquote;
   float: left;
-  margin: $pullquote-margin-top $pullquote-left-margin-right $pullquote-margin-bottom $pullquote-left-margin-left;
+  margin: if-map-get($pullquote-margin-top, $type) $pullquote-left-margin-right if-map-get($pullquote-margin-bottom, $type) $pullquote-left-margin-left;
```

Comment:
Let's add maps for the `$pullquote-left-margin-left`, `$pullquote-left-margin-right`, `$pullquote-right-margin-right`, and `$pullquote-right-margin-left` vars as well please!